lib/src/ui/movies_page.dart

```diff
@@ -112,9 +85,46 @@ class _MoviesPageState extends State<MoviesPage> {
                     widget.bloc.fetchByMovieName(textField);
                   },
                 ),
-              ),
-            ],
+          leading: IconButton(
+            icon: Icon(
+              Icons.view_headline_outlined,
+              color: Colors.purple,
+              size: UiConstants.iconSize,
+            ),
+            onPressed: () {},
```

Comment:
This leading is necessary?

Reply:
Yes, because Kevin has to add our business cards functionality in there